src/Components/PaperEditor/TranscriptsContainer/LabelsList/index.js

```diff
@@ -204,7 +205,6 @@ const LabelsList = (props) => {
 
 LabelsList.propTypes = {
   labels: PropTypes.any,
-  isLabelsListOpen: PropTypes.any,
```

Comment:
`isLabelsListOpen` prop removed from `LabelsList` as it was only used to decide whether or not to show the `LabelsList` in parent component

Reply:
👍